modules/tensor_mechanics/test/tests/torque_reaction/tests

```diff
@@ -1,34 +1,53 @@
 [Tests]
   [./torque_reaction_2D]
+    design = TorqueReaction.md
+    issues = '#6484'
+    requirement = 'The TorqueReaction Postprocessor shall compute the torque for an applied deformation at the end of a beam in 2D'
     type = 'Exodiff'
     input = 'torque_reaction_tm.i'
     exodiff = 'torque_reaction_tm_out.e'
```

Comment:
Does the regolded exodus file still match the analytical solution that is given in the input file / does that analytical solution need to be updated to reflect the increased applied rotation displacement?

Reply:
This test was actually not affected by the change because it doesn't use the `DisplacementAboutAxis` BC. The `torque_reaction_cylinder` test was slightly affected, but it did not list a result in the .i file, and the change was very minor because it was in the small rotation regime.